test/ibmq/runtime/fake_runtime_client.py

```diff
@@ -72,18 +72,23 @@ class BaseFakeRuntimeJob:
 
     _executor = ThreadPoolExecutor()
 
-    def __init__(self, job_id, program_id, hub, group, project, backend_name, params):
+    def __init__(self, job_id, program_id, hub, group, project, backend_name, final_status,
```

Comment:
I personally wouldn't add `final_status` here, as it's not a real job attribute. Even though this is a fake job, it'd be nice to keep it as realistic as possible. But since it is fake, I don't feel too strongly if you do't want to change it.

Reply:
I'd like to keep it since it makes testing easier by setting the job's final status and avoid time.sleep which makes the tests flaky and slow.